xbmc/media/MediaType.cpp

```diff
@@ -75,6 +75,14 @@ MediaType MediaTypes::ToPlural(const MediaType &mediaType)
   return mediaTypeIt->second.plural;
 }
 
+bool MediaTypes::IsCollection(const MediaType &mediaType)
```

Comment:
Name is misleading then. Could we please change this to a more descriptive name (if needed)?